utils/generator/api-model-generator.ts

```diff
@@ -68,8 +68,14 @@ export function writeStatementsToFile(statements: string[]): void {
 export function executeCommand(model: APIModel): APIModel {
   const command = `openapi-generator-cli generate -g typescript-axios --additional-properties=supportsES6=true,useSingleRequestParameter=true --type-mappings=set=Array --skip-validate-spec -o ${model.outputPath} -i ${model.modelPath}`
   log.info(`Starting generating ${model.dirname}`)
-  // TODO: throw an error when command occurs error
-  execSync(command)
+  /**
+   * TODO: Investigate:
+   * openapi-generator-cli requires Java lib.
+   * Java is not installed locally. But system doesn't throw an Error.
```

Comment:
1. We can install java in the gh actions env, it's just a linux box all tools are available
2. We can run inside Docker, probably as well as another option

Reply:
At the moment, we are using option 1: https://github.com/ScaleLeap/selling-partner-api-sdk/blob/master/.github/workflows/generate-models.yml#L29
👍